cfgov/jinja2/v1/_includes/molecules/breadcrumbs.html

```diff
@@ -17,9 +17,13 @@
     <nav class="breadcrumbs" aria-label="Breadcrumbs">
         {{ svg_icon('left') }}
         {% for crumb in breadcrumbs %}
-        <a class="breadcrumbs_link" href="{{ crumb.href if crumb.href else get_protected_url(crumb) }}">
-            {{ crumb.title | e }}
-        </a>
+            {% if crumb.href or crumb.live %}
```

Comment:
This change adds a check to see if `crumb.href` or `crumb.live` exist. The `href` attribute will indicate it's a dictionary with an `href`, the `live` attribute will indicate that it's a page-like object (the first thing `get_protected_url` uses from page-like objects is `page.live`).

Reply:
I dig it! If there's no `live` attribute the [get_protected_url](https://github.com/cfpb/cfgov-refresh/blob/ca9053d66e65ab0f66490fab6f08a4b7354354f8/cfgov/v1/routing.py#L55-L65) function doesn't return a URL so removing the hyperlink for non-`live` breadcrumbs makes sense.